### PR TITLE
Added the npx configure-react commands that help setup react app faster

### PR DESCRIPTION
**Pull request by shaantanu9**
### What does this PR do?

Added the `npx configure-react` commands that help setup react app faster

### Description
Added the npx configure-react commands that help setup react app faster
https://www.npmjs.com/package/configure-react

### Screenshots or Links

npx configure-react

### Additional context
This help any new developer to set the React App Faster and can start learning and implementing things easily

### Checklist:
- [x] Read our [contributing guidelines](../docs/CONTRIBUTING.md).
- [x] Search for duplicates.
- [x] Used an informative name for this pull request.

### Follow-up
- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Review by crescentpartha — CHANGES_REQUESTED** [review]
Require some modifications. @shaantanu9

File: CheatSheets/configure-react-cheatsheet.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+  - [React Commands to Configure React App Faster (Specially for Beginners)](#react-commands-to-configure-react-app-faster-specially-for-beginners)
+  - [Create React Cluster App (Multiple React App in One Project) Build new React App in Second](#create-react-cluster-app-multiple-react-app-in-one-project-build-new-react-app-in-second)
+
+# Configure-React CheatSheet for Developers
+
+
+## React Commands to Configure React App Faster (Specially for Beginners)
+| Command | Description |
+| :-------: | :-----------: |
+| `npx configure-react axios . ` | Install axios and create a folder named `api` and create a file named `axios.js` login.js, register.js, post.js getdata.js |
```
[the remark this answers is not in the document]
Comment (shaantanu9) [reply CheatSheets/configure-react-cheatsheet.md:19]: Yes **. dot is needed** to know that you are in the root folder, similar to git add .

File: CheatSheets/npx-cheetsheet.md [hunk 2]
```diff
@@ -19,6 +19,7 @@ created: 2022-10-24
 | `npx npm-check -u` | Update all project dependencies |
 | `npx browserslist` <br /> OR <br /> `npx browserslist "last 2 versions, not dead, not < 2%"` | Browsers list |
 | `npx create-react-app {name} --template {template}` | Templates can be added according to developer's need, like redux template, <br /> `npx create-react-app my-app --template redux` |
+|`npx sirdeploy github .`| Deploy React App on Github|
```
[the remark this answers is not in the document]
Comment (shaantanu9) [reply CheatSheets/npx-cheetsheet.md:22]: This is the ScreenShot of the `npx sirdeploy`

sirdeploy

And this is the Package Link;-  https://www.npmjs.com/package/sirdeploy

[the remark this answers is not in the document]
Comment (shaantanu9) [reply CheatSheets/npx-cheetsheet.md:22]: sorry, its `npx sirdeploy reactapp` to deploy react app on github will correct in the next commit,

This is the Video of the same https://www.youtube.com/watch?v=fybRnR-fSCk

File: CheatSheets/configure-react-cheatsheet.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+|`npx configure-react refresh . `|Refresh the files and update the files if any changes in the package |
+|`npx configure-react create-cluster <projectname>`|Create Big React Project with Tailwind, React Router Dom, Chakra UI, Context API, Redux, Redux Thunk, Redux Saga, Redux Persist, Utils, Axios |
+|`npx configure-react cluster-app <projectname>`|Create Cluster App with Tailwind, React Router Dom, Chakra UI, Context API, Redux, Redux Thunk, Redux Saga, Redux Persist, Utils, Axios |
+|`npx configure-react ic <package-name>`|Install package in Cluster App |
+|`npx configure-react help`|Display help for command |
+
+
+**[🔼Back to Top](#table-of-contents)**
+
+## Create React Cluster App (Multiple React App in One Project) Build new React App in Second
```
[the remark this answers is not in the document]
Comment (shaantanu9) [reply CheatSheets/configure-react-cheatsheet.md:39]: Correcting this, but the Above section commands can be used in any already build react-app not specifically for cluster app, and that is why to know the root where package.json is present using that.

File: CheatSheets/configure-react-cheatsheet.md [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+Watch this video to create React Cluster App (Multiple React App in One Project) 
+
+[React Cluster](https://www.youtube.com/watch?v=2MO1_mCXuds&t=3s)
```
[the remark this answers is not in the document]
Comment (shaantanu9) [reply CheatSheets/configure-react-cheatsheet.md]: corrected in latest commit, please check

File: CheatSheets/npx-cheetsheet.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (shaantanu9) [reply CheatSheets/npx-cheetsheet.md]: corrected in latest commit, please check

**Review by crescentpartha — APPROVED** [review]
Approved your changes.

**Pull request merged**